### DISCUSSION TITLE
Can we have some features to handle the AUTOMATIC1111 filename or foldername?

### DISCUSSION THREAD

**Discussion by chouti** [adieyal/sd-dynamic-prompts#538] [Ideas]
I am a happy user of dynamic prompts especially with wildcards support.

And I have one workflow combine dynamic prompts and agent scheduler, e.g: Create bunch of schedule in queue, each task was a combination of wildcards prompt. So that I can run over night and get lots of outputs.

But, I'd like to have the ability to identify which file was generated by which wildcards prompt. Currently my output folder was simply the [date], and file name pattern was [model_name]-[sampler]-[steps]-[seed]-[cfg]_[date], I know we can add [prompt] into the filename, but due to the filename length limitation it is not the best way to achieve my goal.

I'm looking for a way to save outputs in a subfolder like [date]\[wildcardname]\filename, or in filename like [date]\[wildcardname]-[model_name]-[sampler]-[steps]-[seed]-[cfg]_[date]

**Comment by adieyal** [comment]
How would you handle a prompt with multiple wildcards? e.g. `__wildcard1__ __wildcard2__ __wildcard3__`? Currently you are able to save the raw prompts to a csv file in case that's useful.
